# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../matrox-mtx/install.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./kernel/src/Makefile_2.6.vanilla	2020-07-27 06:51:30.704553726 +0000
+++ ./kernel/src/Makefile_2.6	2020-07-27 06:51:48.297555004 +0000
@@ -71,8 +71,8 @@
 	$(DEPEND) $(EXTRA_CFLAGS) $(wildcard *.c)
 
 install:
-	mkdir $(INSTALLDIR)
+	mkdir -p $(INSTALLDIR)
 	$(INSTALL) -m 0644 -o root -g root mtx.ko $(INSTALLDIR)/mtx.ko
-	depmod -ae
+	#depmod -ae
 
 endif
